src/merge.py

```diff
@@ -14,9 +14,23 @@ def merge(x: list[int], y: list[int]) -> list[int]:
     i, j = 0, 0
     z = []  # a new list to copy elements into
     # FIXME: fill out the loop so you merge the lists
-    # until one of them is empty
     while i < len(x) and j < len(y):
-        break  # FIXME: you shouldn't just break here
+        if x[i] < y[j]:
+            z.append(x[i])
+            i+=1
+        #elif y[j]>=x[i]:
+            #i+=1
+        else:
+            z.append(y[j])
+            j+=1
+
+    if j== len(y):
+        z=z+x[i:]
+    elif i==len(x):
+        z=z+y[j:]
```

Comment:
If you use `z.extend(x[i:])` and `z.extend(y[j:])` instead of concatenation, you get a slightly faster solution. Your solution will copy all of the elements in `z` to do the concatenation, but the `extend` (because it can just move the other list into `z`) is faster.

Also, you don't strictly need to check which list you need to concatenate. You can just always do both. One of them will be empty, and it is not doing any harm if you extend `z` with it.